geode-wan/src/main/java/org/apache/geode/internal/cache/wan/GatewayReceiverImpl.java

```diff
@@ -138,6 +136,29 @@ public CacheServer getServer() {
     return receiver;
   }
 
+  private boolean tryToStart() {
+    if (!AvailablePort.isPortAvailable(port, AvailablePort.SOCKET,
```

Comment:
This will possibly help but there still is the initial problem where if multiple receivers get the same start port, they all can check available port and all return true before one of them actually permanently binds.  I guess the issue is that we still have to wait 2 minutes if two ports are the same

Reply:
yes, that's possible. As I said, it will be improved by reducing 2 miniute to a few seconds, or other solution in another  ticket.